test/libyul/ewasmTranslationTests/difficulty.yul

```diff
@@ -4,6 +4,6 @@
 // ----
 // Trace:
 // Memory dump:
-//     20: 9999990900000000000000000000000000000000000000000000000000000000
+//     20: 0000000000000000000000000000000000000000000000000000000009999999
```

Comment:
All of these tests would make more sense with having `mstore(0, <some random number>) mstore(32, <some random number>)` at the beginning to clobber the space the polyfill is writing to.

Reply:
I agree. We should improve the ewasm translation tests. I think the memory there should not be initialized with zero's. So we could easily check how many bytes where written by the e.g. polyfills.

Reply:
It should be initialised to zeroes, as it is done so on wasm and evm.

However we should change the tests to explicitly make those slots dirty.

Reply:
This PR still waits for adding these clobbering instructions to these files.

Reply:
@axic Is `0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff` good enough for that or do you think it should actually be something looking more random?

Reply:
So actually instead of `sstore()` we wanted to use `mstore()` for clobbering, however that starts at actual memory 0x40 and the lower 0x40 bytes are used as scratch space in the translators:
```
+++ b/test/libyul/ewasmTranslationTests/origin.yul
@@ -1,9 +1,13 @@
 {
+  mstore(0, 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff)
+  mstore(32, 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff)
   sstore(0, origin())
 }
 // ----
 // Trace:
 // Memory dump:
 //     20: 0000000000000000000000000000000000000000000000000000000033333333
+//     40: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
+//     60: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
 // Storage dump:
 //   0000000000000000000000000000000000000000000000000000000000000000: 0000000000000000000000000000000000000000000000000000000033333333
```

I think we should just remove sstores from this PR and try to figure out clobbering later.